packages/components/src/input/src/Input.css

```diff
@@ -1,32 +1,32 @@
 .o-ui {
-    --o-ui-input-padding: var(--o-ui-sp-3);
+    --o-ui-input-padding: var(--hop-space-inset-squish-md);
     --o-ui-input-padding-with-icon: var(--o-ui-sp-7);
     --o-ui-input-padding-with-button: var(--o-ui-sp-8);
     --o-ui-input-padding-loading: var(--o-ui-sp-8);
-    --o-ui-input-border-color: var(--o-ui-b-alias-mid-break);
-    --o-ui-input-border-radius: var(--o-ui-br-2);
-    --o-ui-input-background-color: var(--o-ui-bg-alias-mid-break);
+    --o-ui-input-border-color: var(--hop-neutral-border);
+    --o-ui-input-border-radius: var(--hop-shape-rounded-md);
+    --o-ui-input-background-color: var(--hop-neutral-surface);
 }
 
 .o-ui-input {
     display: inline-flex;
     position: relative;
     transition: var(--o-ui-focus-ring-transition);
-    color: var(--o-ui-text-alias-primary);
-    width: var(--o-ui-sz-16);
+    color: var(--hop-neutral-text);
+    width: 20rem;
     align-items: center;
 }
 
 .o-ui-input input {
     -webkit-appearance: none;
-    font-family: inherit;
-    font-size: inherit;
+    font-family: var(--hop-body-md-font-family);
+    font-size: var(--hop-body-md-font-size);
     outline: transparent;
     border: none;
     width: 100%;
     background-color: inherit;
-    color: inherit;
-    min-height: calc(var(--o-ui-sz-6) - 2px);
+    color: var(--hop-neutral-text);
```

Comment:
What was wrong with leaving it as inherit since it's already defined in .o-ui-input?
Same with font-family and font-size. Not sure if it's defined elsewhere, but maybe it could also be defined in .o-ui-input. Just a suggestion.